lib/player.rb

```diff
@@ -0,0 +1,11 @@
+class Player
+	attr_accessor :credits, :active
+
+	def sub_credits(sub_creds)
+		if sub_creds == 0
+		  raise StandardError
```

Comment:
I'd suggest you create a custom error and raise it. It will help with creating custom errors.

Reply:
created a custom method for error.